src/test/java/BaseTest.java

```diff
@@ -42,20 +42,20 @@ public void launchBrowser(@Optional String baseURL) {
         driver = new ChromeDriver();
         actions = new Actions(driver);
         // Make webdriver load the pages REALLY slow
-//        WebDriver augmentedDriver = new Augmenter().augment(driver);
-//        ChromiumNetworkConditions networkConditions = new ChromiumNetworkConditions();
-//        networkConditions.setDownloadThroughput(100 * 1024);
-//        networkConditions.setUploadThroughput(500 * 1024);
-//        networkConditions.setLatency(Duration.ofMillis(5000));
-//        ((HasNetworkConditions) augmentedDriver).setNetworkConditions(networkConditions);
+        WebDriver augmentedDriver = new Augmenter().augment(driver);
+        ChromiumNetworkConditions networkConditions = new ChromiumNetworkConditions();
+        networkConditions.setDownloadThroughput(100 * 1024);
+        networkConditions.setUploadThroughput(500 * 1024);
+        networkConditions.setLatency(Duration.ofMillis(5000));
+        ((HasNetworkConditions) augmentedDriver).setNetworkConditions(networkConditions);
         // (comment out above lines to remove throttling)
 
         // Wait for an element to show up for max of X seconds
         // implicitlyWait(Duration.ofSeconds(60) will wait for UP to 60 seconds
         // if element comes up after 1 second, it will move on
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        wait = new WebDriverWait(driver,Duration.ofSeconds(10));
+        wait = new WebDriverWait(driver,Duration.ofSeconds(50));
```

Comment:
a description of the work of the methods must be written above the method, everything else must be deleted